integration_test/cypress/e2e/notebooks/notebooks.ts

```diff
@@ -91,7 +91,7 @@ function executeNotebook(notebookPath: string) {
     'pytest --nbmake --nbmake-timeout=150000 --overwrite "' +
       notebookPath +
       '"',
-    150000
+    160000
```

Comment:
Does it matter if the `--nbmake-timeout` doesn't match this?

Reply:
this was intentional - i wanted the pytest command to fail before the exec command timed out (in order to get stdout)